src/main/resources/application.properties

```diff
@@ -3,27 +3,23 @@ spring.application.name=GabartiScholars-QB-BE
 # Server configuration
 server.port=8080
 
-# Database configuration (will be overridden by environment variables in Docker)
 spring.datasource.url=jdbc:mysql://localhost:3306/mydatabase
 spring.datasource.username=myuser
 spring.datasource.password=secret
 spring.datasource.driver-class-name=com.mysql.cj.jdbc.Driver
```

Comment:
Environment values shouldn't be hardcoded and should be read from `.env` file in general.
and for this specific case, we should use the docker env vars to communicate with the `mysql` docker contianer